Tests are failing if CLN is not available

Especially relevant for Apple silicon users.
```console
______________________________________________________________________________ TestParametric.test_constraints_collector ______________________________________________________________________________

self = <test_parametric.TestParametric object at 0x123649ad0>

    def test_constraints_collector(self):
        from pycarl.formula import FormulaType, Relation
        if stormpy.info.storm_ratfunc_use_cln():
>           import pycarl.cln.formula

tests/pars/test_parametric.py:82: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from . import _config
    
    if not _config.CARL_WITH_CLN:
>       raise ImportError("CLN is not available in the configured carl library! Did you configure carl with -DUSE_CLN_NUMBERS=ON?")
E       ImportError: CLN is not available in the configured carl library! Did you configure carl with -DUSE_CLN_NUMBERS=ON?

../pycarl/lib/pycarl/cln/__init__.py:4: ImportError
_____________________________________________________________________________________ example_building_models_02 ______________________________________________________________________________________

    def example_building_models_02():
    
    
        import stormpy.pars
        if stormpy.info.storm_ratfunc_use_cln():
>           import pycarl.cln as pc

examples/building_models/02-building-models.py:14: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from . import _config
    
    if not _config.CARL_WITH_CLN:
>       raise ImportError("CLN is not available in the configured carl library! Did you configure carl with -DUSE_CLN_NUMBERS=ON?")
E       ImportError: CLN is not available in the configured carl library! Did you configure carl with -DUSE_CLN_NUMBERS=ON?

../pycarl/lib/pycarl/cln/__init__.py:4: ImportError
____________________________________________________________________________________ example_parametric_models_02 _____________________________________________________________________________________

    def example_parametric_models_02():
        # Check support for parameters
        if not config.storm_with_pars:
            print("Support parameters is missing. Try building storm-pars.")
            return
    
        import stormpy.pars
        from pycarl.formula import FormulaType, Relation
        if stormpy.info.storm_ratfunc_use_cln():
>           import pycarl.cln.formula

examples/parametric_models/02-parametric-models.py:21: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from . import _config
    
    if not _config.CARL_WITH_CLN:
>       raise ImportError("CLN is not available in the configured carl library! Did you configure carl with -DUSE_CLN_NUMBERS=ON?")
E       ImportError: CLN is not available in the configured carl library! Did you configure carl with -DUSE_CLN_NUMBERS=ON?

../pycarl/lib/pycarl/cln/__init__.py:4: ImportError
____________________________________________________________________________________ example_parametric_models_01 _____________________________________________________________________________________

    def example_parametric_models_01():
        # Check support for parameters
        if not config.storm_with_pars:
            print("Support parameters is missing. Try building storm-pars.")
            return
    
        import stormpy.pars
        from pycarl.formula import FormulaType, Relation
        if stormpy.info.storm_ratfunc_use_cln():
>           import pycarl.cln.formula

examples/pomdp/01-pomdps.py:23: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    from . import _config
    
    if not _config.CARL_WITH_CLN:
>       raise ImportError("CLN is not available in the configured carl library! Did you configure carl with -DUSE_CLN_NUMBERS=ON?")
E       ImportError: CLN is not available in the configured carl library! Did you configure carl with -DUSE_CLN_NUMBERS=ON?

../pycarl/lib/pycarl/cln/__init__.py:4: ImportError
======================================================================================= short test summary info =======================================================================================
FAILED tests/core/test_multiobjective.py::TestModelChecking::naive_api_double_no_plotting_test - assert 4 == 6
FAILED tests/pars/test_parametric.py::TestParametric::test_constraints_collector - ImportError: CLN is not available in the configured carl library! Did you configure carl with -DUSE_CLN_NUMBERS=ON?
FAILED examples/building_models/02-building-models.py::example_building_models_02 - ImportError: CLN is not available in the configured carl library! Did you configure carl with -DUSE_CLN_NUMBERS=ON?
FAILED examples/parametric_models/02-parametric-models.py::example_parametric_models_02 - ImportError: CLN is not available in the configured carl library! Did you configure carl with -DUSE_CLN_NUMBERS=ON?
FAILED examples/pomdp/01-pomdps.py::example_parametric_models_01 - ImportError: CLN is not available in the configured carl library! Did you configure carl with -DUSE_CLN_NUMBERS=ON?
============================================================================== 5 failed, 385 passed, 2 skipped in 31.01s ==============================================================================
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -38,8 +38,8 @@ endif ()
 # Set configurations
 set(STORM_VERSION ${storm_VERSION})
 # Set number types from Carl
-set_variable_string(STORM_USE_CLN_EA_BOOL STORM_USE_CLN_EA)
-set_variable_string(STORM_USE_CLN_RF_BOOL STORM_USE_CLN_RF)
+set_variable_string(STORM_USE_CLN_EA_BOOL ${STORM_USE_CLN_EA})
+set_variable_string(STORM_USE_CLN_RF_BOOL ${STORM_USE_CLN_RF})
 if (STORM_USE_CLN_EA)
     set(PYCARL_EA_PACKAGE "cln")
 else()

